Lib/test/test_bytes.py

```diff
@@ -802,6 +802,12 @@ def __int__(self):
             with self.assertRaisesRegex(TypeError, msg):
                 operator.mod(format_bytes, value)
 
+        # gh-140939: MemoryError is raised without leaking
```

Comment:
Please move this code into a new test, since `import_helper.import_module()` can skip the test if the module is missing.